angularjs-1.5-basics/gulp-tasks/release.js

```diff
@@ -0,0 +1,90 @@
+'use strict';
+
+var gulp = require('gulp');
+var git = require('gulp-git');
+var bump = require('gulp-bump');
```

Comment:
__[jshint]__
* 'require' is not defined.

<!-- Comment by [SideCI](https://sideci.com) -->